cmd/tailscale/cli/file.go

```diff
@@ -49,6 +52,21 @@ var fileCmd = &ffcli.Command{
 	},
 }
 
+type countingReader struct {
+	io.Reader
+	n uint64
+}
+
+func (c *countingReader) Read(buf []byte) (int, error) {
+	n, err := c.Reader.Read(buf)
+	atomic.AddUint64(&c.n, uint64(n))
+	return n, err
+}
+
+func (c *countingReader) ReadBytes() uint64 {
```

Comment:
... then this is kinda useless. caller can do `n.Load()` just as easily/clearly

Reply:
Done.